### PR TITLE
Added DFN-8-1EP package for MicroChip

### PR DESCRIPTION
**Pull request by svollenweider**
Pull Request related to 
https://github.com/KiCad/kicad-symbols/issues/2732

### CODE REVIEW

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/dfn.yaml [hunk 1]
```diff
@@ -341,6 +341,49 @@ DFN-8-1EP_2x2mm_P0.5mm_EP0.9x1.5mm:
...
+  # ipc_class: 'qfn' # 'qfn_pull_back'
+  #ipc_density: 'least' #overwrite global value for this device.
+  body_size_x:
+    nominal: 3
+    tolerance: 0
+  body_size_y:
+    nominal: 2
+    tolerance: 0
+  overall_height:
+    minimum: 0.8
```
Comment (chschlue) [inline scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/dfn.yaml:358]: Please add nominal.

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/dfn.yaml [hunk 2]
```diff
@@ -341,6 +341,49 @@ DFN-8-1EP_2x2mm_P0.5mm_EP0.9x1.5mm:
...
+    maximum: 1
+  lead_width:
+    minimum: 0.2
```
Comment (chschlue) [inline scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/dfn.yaml:361]: Please add nominal.

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/dfn.yaml [hunk 3]
```diff
@@ -341,6 +341,49 @@ DFN-8-1EP_2x2mm_P0.5mm_EP0.9x1.5mm:
...
+    maximum: 0.3
+  lead_len:
+    minimum: 0.3
```
Comment (chschlue) [inline scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/dfn.yaml:364]: Please add nominal.

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/dfn.yaml [hunk 4]
```diff
@@ -357,19 +357,24 @@ DFN-8-1EP_2x3mm_P0.5mm_EP1.4x1.7mm:
...
     maximum: 0.3
+    nominal: 0.25
   lead_len:
     minimum: 0.3
     maximum: 0.5
+    nominal: 0.4
   EP_size_x:
     minimum: 1.5
     maximum: 1.75
+    nominal: 1.7
```
Comment (chschlue) [inline scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/dfn.yaml:372]: Please remove nominal :)
Or am I missing something?

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/dfn.yaml [hunk 5]
```diff
@@ -357,19 +357,24 @@ DFN-8-1EP_2x3mm_P0.5mm_EP1.4x1.7mm:
...
   EP_size_x_overwrite: 1.7
   EP_size_y:
     minimum: 1.3
-    maximum: 1.55
+    maximum: 1.45
```
Comment (chschlue) [inline scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/dfn.yaml:376]: 1.55 was correct before.

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/dfn.yaml [hunk 6]
```diff
@@ -357,19 +357,24 @@ DFN-8-1EP_2x3mm_P0.5mm_EP1.4x1.7mm:
...
+    nominal: 1.4
```
Comment (chschlue) [inline scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/dfn.yaml:377]: Please remove.

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/dfn.yaml (hunk 5 above)
Comment (svollenweider) [reply scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/dfn.yaml:376]: I am not sure. Is the generator taking the pad size of the IC or the pad size of the footprint?

I realized that 1.55 is the size of the pad on the bottom of the chip, but 1.45 is the maximum size for the pad.

Comment (chschlue) [reply scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/dfn.yaml:375]: Except for EP_size_[xy]_overwrite, all definitions are _package_ dimensions from which the footprint is generated.
BTW, please cross-reference FP and generator PRs.

**Comment by chschlue** [comment]
https://github.com/KiCad/kicad-footprints/pull/2284

Please resolve conflicts.

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/dfn.yaml [hunk 7]
```diff
@@ -341,6 +341,51 @@ DFN-8-1EP_2x2mm_P0.5mm_EP0.9x1.5mm:
...
+    minimum: 0.3
+    maximum: 0.5
+    nominal: 0.4
+  EP_size_x:
+    minimum: 1.5
+    maximum: 1.75
+  EP_size_x_overwrite: 1.7
+  EP_size_y:
+    minimum: 1.3
+    maximum: 1.45
```
Comment (chschlue) [inline scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/dfn.yaml:375]: Please fix

**Review by chschlue — APPROVED** [review]

**Pull request merged**